de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/concept/types/structural/ABeanStructuralElementInstance.java

```diff
@@ -38,6 +52,10 @@
  * @author fisc_ph
  *
  */
+@XmlAccessorType(XmlAccessType.NONE)
+@XmlRootElement
+// Ensure that the sei (by uuid) gets unmarshalled first
+@XmlType(propOrder = {"structuralElementInstance", "name", "parentSeiBean", "categoryAssignments", "children", "superSeis"})

```

Comment:
Why do we need the prop order? I thought the XmlID tag makes the structuralElementInstance property first in line?

Reply:
Good question, I know for a fact that it doesn't work if I delete that line atm. But I will investigate it.

Reply:
Ok so I looked into it and the documentation says nothing about changing the order if using `@XmlID`, instead it can be used together with `@XmlIDREF` to model reference and containment. 
So for testing I uncommented the Annotation in ABeanObject and it didn't have any effect. The reason that it gets unmarshalled first in ABeanObject actually is the naming of the method: "getATypeInstance" which is handled as "aTypeInstance" field and so is first alphabetically ordered. So we could either rename the method or use the propOrder (for which all properties have to be listed).

Reply:
So i removed the `@XmlID` annotations